client/src/components/Form.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { createNewPassword } from "../utils/api";
+
+export default function Form() {
+  const history = useHistory();
+  const [passwordName, setPasswordName] = useState("");
+  const [userdata, setUserdata] = useState("");
+  const [passwordValue, setPasswordValue] = useState("");
+
+  function handleNameChange(e) {
+    setPasswordName(e.target.value);
```

Comment:
```suggestion
    e.preventDefault()
    setPasswordName(e.target.value);
```
Hier sollte das rein, damit der state sich nicht ständig ändert